### ISSUE TITLE
Research and recommend best practice for using quotes around text in .md files

### ISSUE THREAD

**Issue by codyyjxn** [hackforla/website#7565]
### Overview
We need to research and recommend best practice so that we can audit the site to determine which quotes or lack of quotes need to be fixed.

### Details
Here is an example of two text items in the same file. 

https://github.com/hackforla/website/blob/gh-pages/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml?plain

https://github.com/hackforla/website/blob/271a82e89f55bdeeb80677403558228aee2bbe24/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml?plain=1#L224

https://github.com/hackforla/website/blob/271a82e89f55bdeeb80677403558228aee2bbe24/.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml?plain=1#L233

Here are examples from two different project .md files.  Notice that first one one uses a double quote and the second uses a single quote for the same type of link
https://github.com/hackforla/website/blob/53e798a794d43ab507acf68f1ebbabf2a54b9bf0/_projects/food-oasis.md?plain=1#L90

https://github.com/hackforla/website/blob/53e798a794d43ab507acf68f1ebbabf2a54b9bf0/_projects/100-automations.md?plain=1#L53

### Action Items
- [x] Review .md and .yml files in the repository containing quotes and identify:
  - [x] What types of data are currently put in quotes
  - [x] Where the data in quotes is being used (to generate an HTML page, in an issue template, etc.)
- [x] Research best practices for using quotes within .md/.yml files and any other files that pull quoted data from the .md/.yml files
- [x] Write a comment on this issue with a recommendation for how quotes should be used in .md/.yml files, including:
  - [x] Types of data that should be in quotes, if any
  - [x] Whether single or double quotes should be used
  - [x] Brief explanation of your research findings and justification for your recommendations
- [ ] Finalize recommendations with a team lead

### Additional Notes
Some research was already done for this issue and can be found in the comments, but it needs to be compiled into a concise recommendation. Additional research may be required.

### Resources
- This Issue comes from this ER #7540
- [Markdown single quotes](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%27+language%3AMarkdown&type=code)
- [Markdown double quotes](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%22+language%3AMarkdown&type=code)
- [YAML single quotes](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%27+language%3AYAML&type=code)
- [YAML double quotes](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%22+language%3AYAML&type=code)

**Comment by ExperimentsInHonesty** [comment]
There is an issue template for issues and an issue template for ER.  This issue has an ER title, but is written as an issue.  It needs to be written as an ER which outlines the problem, and defines the issues to be made.  It can also include a solution outline or draft.  

What is written above is a mashup of both.  So I recommend you put a pause on this issue.  Create an ER that outlines the issues that will need to be made.  Get sign off on the approach, and then revise this issue to be the epic for the audit issues.

If that make sense go ahead, or if you want to discuss further, I am available on Sunday at the 10am meeting or Monday at 5pm at the dev/pm meeting.

**Comment by ExperimentsInHonesty** [comment]
@codyyjxn There is an issue template for issues and an issue template for ER.  This issue has an ER title, but is written as an issue.  It needs to be written as an ER which outlines the problem, and defines the issues to be made.  It can also include a solution outline or draft.  

What is written above is a mashup of both.  So I recommend you put a pause on this issue.  Create an ER that outlines the issues that will need to be made.  Get sign off on the approach, and then revise this issue to be the epic for the audit issues.

If that make sense go ahead, or if you want to discuss further, I am available on Sunday at the 10am meeting or Monday at 5pm at the dev/pm meeting.

**Comment by HackforLABot** [comment]
Hello @codyyjxn, we appreciate you taking on this issue, however it looks like you're already working on another issue at this time. Please wait until your current issue is merged before taking on another issue. :)

We are going to unassign you from this issue so you can focus on your current issue.

Hfla appreciates you! :)

**Comment by HackforLABot** [comment]
Hi @codyyjxn, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by ExperimentsInHonesty** [comment]
@codyyjxn 
- when you are done drafting this issue.  Add a `ready for prioritization` label to it.  
- I will take a look and see if we are ready to make the epic issue that would use these recommendations.  See plan https://github.com/hackforla/website/issues/7540#issuecomment-2438436178

**Comment by ExperimentsInHonesty** [comment]
@codyyjxn I think you missed what you were supposed to do for this issue.

1. write this issue so that it has action items that achieve the scope listed in the overview 

> We need to research and recommend best practice so that we can audit the site to determine which quotes or lack of quotes need to be fixed.

The current draft of the action items, tell the person to make the changes, which is premature given that no one has done the research and recommendations yet.  I am not asking you to do the research and recommendations, but rather write an issue for someone to do it.

**Comment by codyyjxn** [comment]
> @codyyjxn I think you missed what you were supposed to do for this issue.
> 
> 1. write this issue so that it has action items that achieve the scope listed in the overview
> 
> > We need to research and recommend best practice so that we can audit the site to determine which quotes or lack of quotes need to be fixed.
> 
> The current draft of the action items, tell the person to make the changes, which is premature given that no one has done the research and recommendations yet. I am not asking you to do the research and recommendations, but rather write an issue for someone to do it.

Got it! Thank you Bonnie!

**Comment by codyyjxn** [comment]
@bonniewolfe Hi bonnie I have changed the instructions. I will attend todays meeting as well.

**Comment by codyyjxn** [comment]
@bonniewolfe How can I get some more guidance on this ?

**Comment by daras-cu** [comment]
Hi @codyyjxn 
Bonnie asked me to help you work on this issue. I've rewritten the action items for you, basically this issue should just be to decide on a standard practice for using quotes in .md and .yml files. Once we have agreed on a standard, you'll create a new epic issue to audit files that may need to be changed.

- This is for the next step, but for an idea of what the epic issue will look like you can look at #5248.

**Comment by HackforLABot** [comment]
Hi @DakuwoN, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by DakuwoN** [comment]
Availability: Monday - Sunday after 11AM until 7PM Central Time
ETA: I should have this done by June 9th Monday.

**Comment by HackforLABot** [comment]
@DakuwoN

Please add update using the below template (even if you have a pull request). Afterwards, remove the 'To Update !' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, June 17, 2025 at 12:05 AM PST.</sub>

**Comment by DakuwoN** [comment]
There are a ton of .md and .yml files... any ones in particular?

**Comment by daras-cu** [comment]
Hi @DakuwoN, for this issue you don't have to go through every single .md and .yml file, just take a look at some of them to get an idea of what they are used for and some of the specific instances where quotes appear. The main thing we want you to do is propose a standard for using quotes so that we can go through each of the files in a future issue and make sure they all adhere to the standard. You will probably have to do some research into the .md and .yml formats to see what function quotes serve and if there are documented standards for using them. Then based on your research and how you saw them used within hackforLA's files, write a recommendation for how we should use quotes in general. Here are a couple searches you can use to find some files to skim through:

[https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%22+%27+language%3AYAML&type=code](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%22+%27+language%3AYAML&type=code)
[https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%22+%27+language%3AMarkdown+&type=code](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+%22+%27+language%3AMarkdown+&type=code)

On a cursory look, I would expect that there are at least some instances where quotes are definitely necessary, like when text contains symbols with meanings within Markdown, such as "#". Also consider whether there is a functional difference between single and double quotes or if we can simply choose one or the other for consistency.

Hopefully this helps, let me know if you need more guidance as you start to research.

**Comment by DakuwoN** [comment]
Here are recommendations for using quotes in Markdown (.md) and YAML (.yaml/.yml) files:

### Markdown (.md)
- **Quotes in Text**: Use straight double quotes "like this" or single quotes 'like this' for quoting text, following your style guide or English conventions.
- **Code Blocks/Inline Code**: Use backticks (`) for inline code or code blocks, not quotation marks.
- **Blockquotes**: For quoting other sources, use the Markdown blockquote syntax with > at the beginning of the line.

**Example:**
```markdown
> "This is a quote from someone."
Here is some `inline code`.
```

### YAML (.yml/.yaml)
- **Unquoted Strings**: Use unquoted strings when the value is simple (no special characters, no leading/trailing spaces, no reserved YAML keywords).
- **Single Quotes ('')**: Use single quotes for strings that contain double quotes or special characters, or when you need the string to be interpreted literally.
- **Double Quotes ("")**: Use double quotes when you need to include special characters (like \n for newline), escape sequences, or interpolation.
- **Always Quote**: Always quote values containing colons (:), leading/trailing spaces, or reserved words (e.g., yes, no, null).
- **Consistency**: Stick to one style throughout the file/project for consistency.

**Example:**
```yaml
title: "Hack for LA's Website"
description: 'This YAML file uses single and double quotes appropriately.'
path: /some/path
note: "Use quotes if the string contains: colon, #, or special chars."
```

#### Summary Table

| When to Quote           | Use Single Quotes         | Use Double Quotes         |
|------------------------ |--------------------------|--------------------------|
| Simple string           | Not needed               | Not needed               |
| Contains single quote   | Use double quotes        | Use double quotes        |
| Contains double quote   | Use single quotes        | Escape double quote (\") |
| Needs escape sequences  | Not supported            | Supported                |
| Reserved word/colon     | Either                   | Either                   |

If your repo has a specific style guide, follow that. Otherwise, the above are widely accepted conventions. If you want to enforce this in your repository, consider using a linter like yamllint for YAML files.

**Comment by DakuwoN** [comment]
In YAML and Markdown (.md) files, using quotes around data depends on the type of data and how you want it to be interpreted:

**YAML:**

- **Strings with special characters**: Use quotes if a string contains special characters (e.g., : { } [ ] , & * # ? | - < > = ! % @), or starts with a number, or includes spaces.
  - Example:  
    `"Hello, world!"`  
    `'2025-07-07'`
- **Boolean or null-like values**: Use quotes to avoid automatic type conversion (e.g., yes, no, on, off, null, true, false).
  - Example:  
    `"yes"` (interpreted as a string, not boolean)
- **Strings with leading zeros**: Use quotes to prevent YAML from interpreting them as numbers.
  - Example:  
    `"007"`
- **Empty strings**: Use quotes for clarity or to explicitly define an empty string.
  - Example:  
    `""`

You can use either single (' ') or double (" ") quotes in YAML. Double quotes allow for escape sequences (like \n), while single quotes do not.

**Markdown (.md):**

- Quotes are rarely required for data in Markdown, since it's primarily a markup language for text. If you're embedding code, configuration, or YAML front matter, follow the YAML rules above.

**Summary Table:**

| Type of data              | Use quotes?         | Example              |
|---------------------------|---------------------|----------------------|
| Special characters        | Yes                 | "foo:bar"            |
| Boolean-like strings      | Yes                 | "no"                 |
| Leading zeros             | Yes                 | "0123"               |
| Numbers (actual numbers)  | No                  | 123                  |
| Simple words              | No (optional)       | hello                |
| Empty string              | Yes                 | ""                   |

**Tip:**  
When in doubt, use quotes in YAML to ensure your data is interpreted as a string. For Markdown, only use quotes if you want to display them as part of the text or in code blocks.

**Comment by DakuwoN** [comment]
There is no universal standard for using single or double quotes in Markdown (.md) or YAML (.yml/.yaml) files—both are valid, but conventions may vary by project.

**Markdown (.md):**
- Quotes are typically used in code blocks, frontmatter, or inline code, and both single and double quotes are acceptable.
- Consistency is most important. There is no strict rule enforced by Markdown itself.

**YAML (.yml/.yaml):**
- Both single and double quotes are allowed for strings.
  - Single quotes: 'string'
  - Double quotes: "string"
- Use quotes if your string contains special characters, spaces at the beginning/end, or YAML-reserved characters (like :, ?, -, etc.).
- Double quotes allow for escape sequences (e.g., \n, \t), while single quotes do not.

**Best Practice:**  
Follow the convention already used in your project for consistency. If the repo has a linter (like yamllint or prettier) or a style guide, check those for specific rules.

**Comment by DakuwoN** [comment]
Hello, so I used the web and also Github Copilot to help with my research results. They both gave similar results, the web was a little more hard to follow as so many different people had different opinions on this matter. It seems to be very subjective as when to use quotes or no quotes as well as single quotes or double quotes, it's a lot of this and that and different tradeoffs/arguments it appears. However, we could follow some traditional conventions that are used as best practices, and make this consistent throughout the repository. They are posted above. It could really depend on project needs and how a particular team wants to implement these concepts.

**Comment by DakuwoN** [comment]
@daras-cu

**Comment by HackforLABot** [comment]
@DakuwoN

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, July 29, 2025 at 12:09 AM PST.</sub>

**Comment by DakuwoN** [comment]
I am waiting for feedback....

**Comment by HackforLABot** [comment]
Hi @michael-4, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by HackforLABot** [comment]
Hi @lastCoyotes, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by lastCoyotes** [comment]
i. Availability: Mon - Thurs 12p-6pm. (Will be completely unavailable from 1/11 - 1/24) 
ii. ETA: Tuesday evening at the latest.

**Comment by lastCoyotes** [comment]
After giving a look at all the files containing quotes and double quotes, I noticed that all of the .md text with quotes to have concern over were in YAML front-matter, so I will be referring to the styling in them the same way.

The current data given in quotes persists of:
- Path Variables
- Alt and Label text
- URLs
- multi-line JavaScript strings to be passed into github-script actions

According to [this blog post](https://yamlscript.org/blog/2025-07-20/yaml-best-practices/) from one of the developers of YAML, all of these values listed above can actually be considered "plain scalars." 

It's absolutely not necessary to give the contents quotes unless:

- It is or starts with a YAML special word (true, false, null, yes, no, ...)
- It contains characters that start a YAML construct ( !, @, #, %, &, *, |, >, ,, {, }, [, ], >, ?, ', ", a backtick or whitespace ) or would trigger special parsing ( :, #)
- You want to preserve leading or trailing whitespace. (e.g. "   keep spaces   ")

However, if things were to be quoted, Ingy döt Net states that you should only use double quotes if you need to escape something. 

I wanted to raise this example not provided in the initial issue, because this one still shows inconsistent styling within the same file, yet also displaying that the quotes aren't strictly necessary in its case as well.

https://github.com/hackforla/website/blob/2b02e882a4e7bf7dbf47e8e5a06e5277158ae487/.github/workflows/set-pr-labels.yaml#L16

https://github.com/hackforla/website/blob/2b02e882a4e7bf7dbf47e8e5a06e5277158ae487/.github/workflows/set-pr-labels.yaml#L45

No quotes are required on line 45. The value contains neither spaces nor any YAML-reserved characters. Additionally, the ${{ … }} syntax is opaque to YAML (it’s processed later by GitHub Actions), so it's perfectly fine to be without quotes.

With that aside, what should we definitely put within quotes?

Exploring the official [YAML parser](https://github.com/yaml/yaml-reference-parser), the parser will automatically type the scalar, so if you want to leave it as a string, definitely quote it (single or double, but preferably single)

It will resolve the trimmed value as null, boolean, integer, float, or timestamp (in this order). If it cannot be resolved into any of these, it will remain a string (!!str)

| Raw plain scalar        | Parsed value / tag                |
|-------------------------|-----------------------------------|
| `true`                  | `true` / `!!bool`                 |
| `0xFF`                  | `255` / `!!int`                   |
| `3.14`                  | `3.14` / `!!float`                |
| `2001-12-14`            | `2001-12-14` / `!!timestamp`      |
| `https://x.com/foo#bar` | `"https://x.com/foo"` / `!!str` *  |
| `C:\temp`               | `"C:\\temp"` / `!!str`            |
| `yes`                   | `true` / `!!bool`                 |
| `yesss`                 | `"yesss"` / `!!str`               |

\* Everything after the '#' is treated as a comment, so only the portion before it is kept.


Sources:

[YAML Official Blog](https://yamlscript.org/blog/2025-07-20/yaml-best-practices/)
[YAML Parser](https://github.com/yaml/yaml-reference-parser)

**Comment by michael-4** [comment]
I reviewed the use of quotes in the repository’s .md and .yml files. Nearly all Markdown files are containers for YAML frontmatter with metadata like URLs, titles, descriptions, and image alt text. One exception I find to this is a few Markdown files use quotes in GitHub Actions for automated issue comments, but this was a small number of files.

**Types of data that should be in quotes, if any**
Standards for both .md and .yml generally do not require the use of quotations at all. Markdown text is already plain text, and YAML standard does not actually require strings to be quoted. However, there are a few cases where quotes can be needed in YAML:
- When a string contains special characters (e.g., @, ?, <>, = etc.)
- When a string can be interpreted as a boolean (e.g., yes, no, on, off)
- When a string can be interpreted as a number (e.g., 001 or 1.0)
Therefore one approach can be to remove the use of all quotes unless strictly necessary. However, this relies on authors correctly identifying when quotes are needed, which can introduce bugs. Using quotes by default as a safety measure reduces the need to evaluate for every string whether a quote is needed or not.

**Single vs double quotes**
Next I've evaluated the pros and cons of using single vs. double quotes. The truth is that both are equally valid and general consensus is that it comes down to personal preference (with the most important thing being uniformity and consistency across a repository).
The most compelling argument is actually the number of occurrences for each one within the existing repository. Filtering for .md and .yml files only, there are 178 files in the repo using single quotes and 50 files using double quotes. Logistically the amount of labor to implement single quotes as the standard would be less intensive.

Aside from that the minor differences between each option are as follows:
Single quotes:
- Work for all scenarios except when you want to incorporate escape characters (e.g. \n for a new line)
- The only special character to handle is the single quote itself (')
- Stylistically may be preferred by some for short strings, but can be less effective for longer natural language sentences where a single quote is more likely to appear as part of the string itself.
Double quotes:
- Allows for escape sequences which can be situationally useful
- Special characters to handle within double quotes are the double quotes (") and backslash (\)
- Best for long form english text because it allows for the common usage of the single quote as part of the string without requiring any special exception.

Given the above, there can be a case for double quotes having more benefits (allows for escape sequences and better for longer natural language sentences). However, the majority of use cases in the HFLA repository do not fit into either of those categories. In fact, a search for backslashes in .md and .yml files found that there are no escape sequences currently in use.

**Conclusion:**
In Markdown content itself, quotes are purely stylistic and not syntactically meaningful. The primary concern is YAML frontmatter and standalone .yml configs. Defaulting to no quotes unless strictly necessary is a valid and clean option but comes with challenges especially across a repository maintained by many contributors. The risk of type misinterpretation can outweigh the minimal visual simplification it would provide.
Using quotations accounts for most edge cases safely and protects against bugs and type coercion. The use of single quotes specifically would align with the majority of existing files and would therefore be the most logical approach to take if the goal is to unify the files in the repository.
**Recommendation: use single quotes by default for all YAML/.md frontmatter and use double quotes only when escape sequences or apostrophes are required in more long form text.**

**Comment by daras-cu** [comment]
@michael-4 @lastCoyotes thanks for doing research for this issue. For the future, please make sure nobody else is assigned to an issue before you assign yourself, and also when you take up an issue make sure you move it from the Prioritized Backlog to In progress. Since this involved research you both did independently, it's ok and you can both get credit for working on this issue.

It looks like you came up with slightly different recommendations. Both of you mentioned that quotes are not usually necessary unless there are special characters or we want to treat a different data type as a string. @michael-4 recommended using single quotes as a safety measure, but I think this would involve a huge amount of work to add them to each string in all our existing files. Looking at the files that contain quotes currently, it looks like most of them can likely be removed, but it is a relatively small number of lines per file (look at any of the project .md files as an example). Depending on how we decide to structure the issues, we could potentially need one issue per file line that needs to be changed, so the less changes the better.

It seems like you both agree single quotes are preferable as we do not have any escape sequences currently, and that seems to be what is most common in our files already. Double quotes must be used when there are escape characters, but it appears there are none currently in any of the files and I can't think of a situation where we would need to use them in the future.

So to summarize, our Markdown and YAML files should contain single quotes only around strings containing special characters or other data types i.e. numbers or Booleans that we want to treat as strings. This will involve removing some existing unnecessary quotes and possibly changing some double quotes to single.

Let me know what you both think and if I'm overlooking anything you found in your research.

**Comment by HackforLABot** [comment]
@lastCoyotes, @michael-4

Please add update using the below template (even if you have a pull request). Afterwards, remove the 'status: To Update !' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, January 19, 2026 at 11:09 PM PST.</sub>

**Comment by HackforLABot** [comment]
@lastCoyotes, @michael-4

Please add update using the below template (even if you have a pull request). Afterwards, remove the 'status: 2 Weeks Inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Monday, January 26, 2026 at 11:18 PM PST.</sub>

**Comment by michael-4** [comment]
@daras-cu 

Thanks for your feedback. I don't think you are missing anything and I agree with your assessment. The amount of lines per file that contain quotes is relatively low, so it  would create fewer issues to remove the unnecessary quotes rather than add quotes to every string.

Let me know if any further action is needed on this issue before we can close it

**Comment by daras-cu** [comment]
@michael-4 sounds good, thanks for confirming. I'll go ahead and close this issue as completed, @lastCoyotes if you have any other notes you can leave a comment on the original ER #7540

**Comment by HackforLABot** [comment]
This issue was reopened because it did not have any of the following:
- A linked PR,
- An `Ignore` label
- A `non-PR contribution` label

**Comment by daras-cu** [comment]
Sorry just realized the bot reopened this after I tried to close it, re-closing it now so you can move on to your next issue.
